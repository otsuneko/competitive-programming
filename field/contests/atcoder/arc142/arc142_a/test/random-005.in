578685203 1656624
