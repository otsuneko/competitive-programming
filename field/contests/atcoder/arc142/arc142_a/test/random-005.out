1656624 4266561 1656624
16566240 4266561 1656624
165662400 4266561 1656624
4266561 1656624 4266561
42665610 1656624 4266561
426656100 1656624 4266561
6
